src/bufrlib.h.in

```diff
@@ -1,13 +1,11 @@
 /** @file
- *  @brief Define signatures to enable a number of BUFRLIB
- *  subprograms to be called directly from C application programs.
+ *  @brief Define signatures to enable a number of NCEPLIBS-bufr subprograms to be called
+ *  from within the C portion of the library.
```

Comment:
"a number of NCEPLIBS-bufr subprograms?" Can we be more specific?

Reply:
We could, but it would be a long list of Fortran and C routines.

Reply:
And do they have nothing in common other than the fact they are in NCEPLIBS-bufr? Were they selected randomly?

Reply:
What they have in common is that they're all called from within C code inside of the library.  Some of them are native C so they only need prototypes defined, while others are native Fortran so they need wrappers defined.